pkg/services/services.go

```diff
@@ -46,11 +46,33 @@ func GenerateManifest(kubeletArgsFromIgnition map[string]string, vxlanPort strin
 		hybridOverlayConfiguration(vxlanPort, debug),
 		kubeProxyConfiguration(debug),
 	}
+	if platform == config.AzurePlatformType && ccmEnabled {
+		*services = append(*services, azureCloudNodeManagerConfiguration())
+	}
 	// TODO: All payload filenames and checksums must be added here https://issues.redhat.com/browse/WINC-847
 	files := &[]servicescm.FileInfo{}
 	return servicescm.NewData(services, files)
 }
 
+// azureCloudNodeManagerConfiguration returns the service specification for azure-cloud-node-manager.exe
+func azureCloudNodeManagerConfiguration() servicescm.Service {
```

Comment:
Does it make sense to generalize this, so that it can be extended as more platforms use CCM? I'm thinking of something like:
```go
func getCloudNodeManagerConfiguration(serviceName, servicePath) servicescm.Service {
   ...
}
```
where the `windows.AzureCloudNodeManagerServiceName` and  `windows.AzureCloudNodeManagerPath` are the parameter values in this case.

Reply:
Each platform will handle its integration with CCM differently,

Reply:
A bit confused about this. Are we going to do this for other platforms then?

Reply:
No, we will not